app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Users Index action</h1>
\ No newline at end of file
+<h1>Users Index action</h1>
+
+<% @users.each do |person| %>
+  <div class="user__post">
+    <%= image_tag "#{person.photo}" %>
+
+  <div class="user__post__info">
+    <h3><%= person.name%></h3>
+    <div class="post__num">
+    <p> Number of posts: <%= person.posts_counter %></p></div>
+  </div>
+  </div>
+<% end %>
```

Comment:
- [ ] Based on the requirements of this project, we should be able to route to the user show page(users/:id) by clicking on the user. Please kindly rectify :wrench:  this issue.